src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -73,15 +69,63 @@ public void storeMessage(Message message) {
       } catch (Exception e) {
         System.err.println("Error reading message.");
         System.err.println(entity.toString());
-        e.printStackTrace();
+        throw e;
```

Comment:
Out of curiosity why did you decide to throw this Exception?

Reply:
I just wanted to give something the original function could catch. Now that I think about it, I can just handle the exception in just the helper function.

Reply:
With this still in here, I would expect this to complain about not marking the function as throwing an exception? But yeah I think you can just get rid of it